certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,965 @@
+              "items": {
+                "$ref": "#/definitions/APISchemaResponse"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/CSSAPI/v2/{entityName}/Update": {
+      "post": {
+        "tags": [
+          "Core Actions"
+        ],
+        "summary": "Update record(s) for a specified entity",
+        "description": "Update record(s) for a specified entity/area with specific criteria and a TriggerWorkflow as flag to run workflow after the update request succeeds or not. Maximum number of records that can be updated is 500 per call.",
+        "operationId": "CSSAPI_UpdateEntity",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "entityName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Table name to update record(s) in.",
+            "x-ms-summary": "Entity Name",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "CSSAPI_EntityListUpdate",
+              "value-path": "EntityNameForAPI",
+              "value-title": "EntityNameDisplay"
+            }
+          },
+          {
+            "name": "update",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIUpdateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/APIUpdateResponse"
+            }
+          }
+        }
+      }
+    },
+    "/CSSAPI/v2/EntityList": {
+      "get": {
+        "tags": [
+          "Discovery"
+        ],
+        "summary": "Pull list of entities that API is allowed to work with",
+        "description": "Returns the list of names and three flagged properties indicating whether it can be edited, updated, or viewed/queried.",
+        "operationId": "CSSAPI_EntityList",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "purpose",
+            "in": "query",
+            "required": false,
+            "x-ms-summary": "Purpose",
+            "description": "Purpose of List (Select/Insert/Update)",
+            "type": "string",
+            "enum": [
+              "select",
+              "insert",
+              "update"
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/APIEntityListResponse"
+              }
+            }
+          },
+          "400": {
+            "description": "BadRequest",
```

Comment:
@nizarhandal please make descriptions for response codes here look like a sentence. E.g. `Bad Request`, `Http Version Not Supported`.